helm_deploy/licences/templates/_envs.tpl

```diff
@@ -28,6 +28,10 @@ env:
         name: dps-rds-instance-output
         key: database_password
 
+  - name: DB_SSL_ENABLED
+    value: {{ .Values.env.DB_SSL_ENABLED }}
+    value: "true"
```

Comment:
don't understand this but might have seen it before - is this a helm specific thing? 

would have thought as it produces yaml the first value would always be overriden with true?

Reply:
No idea.  
Just copied what had been used for Redis since it seemed about right.
I assumed that the first 'value' fell through to the second if it wasn't present.  
That was just a wild guess based on what someone else had written.
Think I'd better find out what it really does - if anything!

Reply:
I simplified this to 
```